GUIDELINES.rst.txt

```diff
@@ -0,0 +1,589 @@
+Guidelines and Coding Conventions
+=================================
+
+Import
+------
+
+The following import conventions are used throughout the NumPy source
+and documentation::
+
+   import numpy as np
+   import matplotlib as mpl
+   import matplotlib.pyplot as plt
+
+Do not abbreviate ``scipy``. There is no motivating use case to
+abbreviate it in the real world, so we avoid it in the documentation
+to avoid confusion.
+
+In our examples, we import the ``data`` submodule instead of its function,
+such that the image can be changed easily::
+
+    from skimage import data
+
+    img = data.coins()
```

Comment:
@soupault Please, read my comment: https://github.com/scikit-image/scikit-image/issues/2538#issuecomment-295104651 and add your point of view :)